opencti-platform/opencti-front/src/components/dataGrid/components/DataTableHeaders.tsx

```diff
@@ -175,7 +175,7 @@ const DataTableHeaders: FunctionComponent<DataTableHeadersProps> = ({
                 orderAsc={!!orderAsc}
               />
             ))}
-          <span style={{ width: 42 }} />
+          <span style={{ width: 'calc( 42px + 12px )' }} />
```

Comment:
it's for decomposing so the code is understandable ?
otherwise `width: '56px'` would be enough I guess :)

Reply:
this messes up with all the vertical alignments